### PR TITLE
en,zh: Fix broken link and modify description about TiDBMonitor

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Fix broken link and modify description about TiDBMonitor

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ kubectl port-forward -n ${namespace} svc/${cluster_name}-prometheus 9090:9090 &>
 
 ### 设置 kube-prometheus 与 AlertManager
 
-在部分情况下，你可能需要 TidbMonitor 同时获取 Kubernetes 上的监控指标。你可以通过设置 `TidbMonitor.Spec.kubePrometheusURL` 来使其获取 [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics。
+Grafana 内置了 NodeInfo 与 PodInfo 监控面板，用于查看 Kubernetes 上的监控指标。如果需要，需要先手动部署 [kube-prometheus](https://github.com/coreos/kube-prometheus)，然后通过设置 `TidbMonitor.Spec.kubePrometheusURL` 来使其获取 kube-prometheus metrics。
```
Comment (DanielZhangQD) [inline zh/monitor-a-tidb-cluster.md:132]:
```suggestion
TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。如果需要查看，需要先手动部署 Kubernetes 集群监控，可以参考 [kube-prometheus](https://github.com/coreos/kube-prometheus)，然后通过设置 `TidbMonitor.spec.kubePrometheusURL` 来获取 Kubernetes 监控数据。
```

File: en/monitor-a-tidb-cluster.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ You can also set `spec.prometheus.service.type` to `NodePort` or `LoadBalancer`,
 
 ### Set kube-prometheus and AlertManager
 
-In some cases, TidbMonitor needs to obtain the monitoring metrics on Kubernetes. To obtain the [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics, configure `TidbMonitor.Spec.kubePrometheusURL`.
+There are NodeInfo and PodInfo dashboards in TidbMonitor Grafana, in order to obtain the monitoring metrics on Kubernetes. If you need, you should deploy Kubernetes monitor manually, reference [kube-prometheus](https://github.com/coreos/kube-prometheus), then configure `TidbMonitor.Spec.kubePrometheusURL` to obtain kube-prometheus metrics.
```
Comment (qiancai) [inline en/monitor-a-tidb-cluster.md:134]:
```suggestion
Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. To use TidbMonitor Grafana to view these monitoring metrics, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus) to manually deploy Kubernetes cluster monitoring, and then set the ` TidbMonitor.spec.kubePrometheusURL` to get Kubernetes monitoring data.
```

**Review by qiancai — COMMENTED** [review]
@KanShiori PTAL Thanks.

File: zh/monitor-a-tidb-cluster.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ kubectl port-forward -n ${namespace} svc/${cluster_name}-prometheus 9090:9090 &>
 
 ### 设置 kube-prometheus 与 AlertManager
 
-在部分情况下，你可能需要 TidbMonitor 同时获取 Kubernetes 上的监控指标。你可以通过设置 `TidbMonitor.Spec.kubePrometheusURL` 来使其获取 [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics。
+TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。如果需要使用 TidbMonitor Grafana 查看这些监控指标，请先参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 手动部署 Kubernetes 集群监控，然后设置 `TidbMonitor.spec.kubePrometheusURL` 获取 Kubernetes 监控数据。
```
Comment (KanShiori) [inline zh/monitor-a-tidb-cluster.md:132]:
```suggestion
TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。如果需要使用 TidbMonitor Grafana 查看这些监控指标，手动部署 Kubernetes 集群监控，可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 。部署 Kubernetes 集群监控后，设置 `TidbMonitor.spec.kubePrometheusURL` 获取 Kubernetes 监控数据。
```

Comment (qiancai) [inline zh/monitor-a-tidb-cluster.md]:
```suggestion
TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。

如果需要使用 TidbMonitor Grafana 查看这些监控指标，请进行以下操作：

1. 手动部署 Kubernetes 集群监控。

    Kubernetes 集群监控有多种部署方式，如果要使用 kube-prometheus 部署, 可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 。
    
2. 设置 `TidbMonitor.spec.kubePrometheusURL` 获取 Kubernetes 监控数据。
```

File: zh/monitor-a-tidb-cluster.md [hunk 4]
```diff
@@ -129,7 +129,7 @@ kubectl port-forward -n ${namespace} svc/${cluster_name}-prometheus 9090:9090 &>
 
 ### 设置 kube-prometheus 与 AlertManager
 
-在部分情况下，你可能需要 TidbMonitor 同时获取 Kubernetes 上的监控指标。你可以通过设置 `TidbMonitor.Spec.kubePrometheusURL` 来使其获取 [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics。
+TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。如果需要使用 TidbMonitor Grafana 查看这些监控指标，需要手动部署 Kubernetes 集群监控，可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 。部署 Kubernetes 集群监控后，设置 `TidbMonitor.spec.kubePrometheusURL` 获取 Kubernetes 监控数据。
```
Comment (qiancai) [inline zh/monitor-a-tidb-cluster.md:132]:
```suggestion
TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。

如需在 TidbMonitor Grafana 中查看这些监控指标，请进行以下操作：

1. 手动部署 Kubernetes 集群监控。

    Kubernetes 集群监控有多种部署方式。如果要使用 kube-prometheus 部署, 可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 。

2. 设置 `TidbMonitor.spec.kubePrometheusURL` 获取 Kubernetes 监控数据。
```

File: en/monitor-a-tidb-cluster.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ You can also set `spec.prometheus.service.type` to `NodePort` or `LoadBalancer`,
 
 ### Set kube-prometheus and AlertManager
 
-In some cases, TidbMonitor needs to obtain the monitoring metrics on Kubernetes. To obtain the [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics, configure `TidbMonitor.Spec.kubePrometheusURL`.
+Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. To use TidbMonitor Grafana to view these monitoring metrics, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus) to manually deploy Kubernetes cluster monitoring, and then set the `TidbMonitor.spec.kubePrometheusURL` to obtain Kubernetes monitoring data.
```
Comment (qiancai) [inline en/monitor-a-tidb-cluster.md:134]:
```suggestion
Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. 

To view these monitoring metrics in TidbMonitor Grafana, take the following steps:

1. Deploy Kubernetes cluster monitoring manually.

   There are multiple ways to deploy Kubernetes cluster monitoring. If you are using kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).

2. Set the `TidbMonitor.spec.kubePrometheusURL` to obtain Kubernetes monitoring data.
```

**Review by qiancai — COMMENTED** [review]
@KanShiori PTAL

File: zh/monitor-a-tidb-cluster.md [hunk 6]
```diff
@@ -129,7 +129,15 @@ kubectl port-forward -n ${namespace} svc/${cluster_name}-prometheus 9090:9090 &>
...
 ### 设置 kube-prometheus 与 AlertManager
 
-在部分情况下，你可能需要 TidbMonitor 同时获取 Kubernetes 上的监控指标。你可以通过设置 `TidbMonitor.Spec.kubePrometheusURL` 来使其获取 [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics。
+TidbMonitor Grafana 默认内置了 Nodes-Info 与 Pods-Info 监控面板，用于查看 Kubernetes 对应的监控指标。
+
+如需在 TidbMonitor Grafana 中查看这些监控指标，请进行以下操作：
+
+1. 手动部署 Kubernetes 集群监控。
+
+    Kubernetes 集群监控有多种部署方式。如果要使用 kube-prometheus 部署, 可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus) 。
```
Comment (qiancai) [inline zh/monitor-a-tidb-cluster.md:138]:
```suggestion
    Kubernetes 集群监控有多种部署方式。如果要使用 kube-prometheus 部署, 可以参考 [kube-prometheus 文档](https://github.com/coreos/kube-prometheus)。
```

**Review by qiancai — APPROVED** [review]
LGTM

File: en/monitor-a-tidb-cluster.md [hunk 7]
```diff
@@ -131,7 +131,15 @@ You can also set `spec.prometheus.service.type` to `NodePort` or `LoadBalancer`,
...
 ### Set kube-prometheus and AlertManager
 
-In some cases, TidbMonitor needs to obtain the monitoring metrics on Kubernetes. To obtain the [kube-prometheus](https://github.com/coreos/kube-prometheus) metrics, configure `TidbMonitor.Spec.kubePrometheusURL`.
+Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. 
+
+To view these monitoring metrics in TidbMonitor Grafana, take the following steps:
+
+1. Deploy Kubernetes cluster monitoring manually.
+
+   There are multiple ways to deploy Kubernetes cluster monitoring. If you are using kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).
```
Comment (qiancai) [inline en/monitor-a-tidb-cluster.md:140]:
```suggestion
   There are multiple ways to deploy Kubernetes cluster monitoring.  To use kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**